azero/contracts/most/lib.rs

```diff
@@ -708,6 +722,29 @@ pub mod most {
             Ok(())
         }
 
+        /// Sets address of the account that receives rewards on behalf of the committee member.
+        ///
+        /// Can only be called by an account that was a committee member in `committee_id`.
+        /// All the unpaid rewards will then be paid out to this account address when `payout_request` is called.
+        /// Comittee members that have set the `payout_account` do not need to call this method again when the committee changes, unless they want to change the payouts account.
+        #[ink(message)]
+        pub fn set_payout_account(
```

Comment:
I think a simpler approach would be to just add an argument to `payout_rewards`, which would be a target account of the payout.

With the current approach we add additional message + handling fees in relayer will need to call two separate methods instead of just one.

Reply:
we can't - AFAIK payout_rewards is callable by anyone